cclient/response/get_balances.h

```diff
@@ -12,7 +12,7 @@ extern "C" {
 typedef struct {
   int_array balances;
   int milestoneIndex;
-  const address_t milestone;
+  const flex_trit_p milestone;
```

Comment:
`flex_trit_p` is a pointer, not sure if marking it as const is useful here? (as the data itself isn't const-protected)